core/common/src/test/java/alluxio/underfs/options/CreateOptionsTest.java

```diff
@@ -19,7 +19,10 @@
 import alluxio.security.group.provider.IdentityUserGroupsMapping;
 import alluxio.util.CommonUtils;
 
-import org.junit.Assert;
+import static org.junit.Assert.assertEquals;
```

Comment:
static imports go in their own section before all other imports, e.g. https://github.com/Alluxio/alluxio/blob/master/core/server/worker/src/test/java/alluxio/worker/block/BlockWorkerTest.java